src/components/App.jsx

```diff
@@ -1,10 +1,10 @@
 
 import { Routes, Route } from "react-router-dom"
 import { useState } from "react"
+import '../App.css'
```

Comment:
You have declared article too high in the project, you only use it in the articles component so there is no need to delcare it here to only pass it down to the articles component